src/pages/admin/training/list/page.tsx

```diff
@@ -17,8 +18,11 @@ export class ListTrainingPage extends React.Component<IProps, {}> {
   public render() {
     return (
       <div>
-        <TrainingTableComponent trainingList={this.props.trainingList}/>
-        <Link to={adminRouteEnums.training.edit}>Go to training Edit</Link>
+        <h1 className="text-center">Active Trainigs</h1>
+        <AutoSizer disableHeight={true}>
+          {({ width }) => <TrainingTableComponent width={width} trainingList={this.props.trainingList} />} 
+        </AutoSizer>
+        <Link to={adminRouteEnums.training.edit}>Go to training Edit</Link>  
```

Comment:
Yes, I know that you keep as before. But it has no sense to go to edit from here if now each row has the navigation to its own edit page by Id. That's why I say you to change by "Go to dashboard" or even remove it.